langkit/regexes.py

```diff
@@ -38,19 +38,40 @@ def wrappee(text):
 _registered = False
 
 
-def _register_udfs():
-    global _registered
-    if _registered:
+def _unregister_metric_udf(old_name: str, namespace: Optional[str] = ""):
+    from whylogs.experimental.core.udf_schema import _multicolumn_udfs
+
+    if _multicolumn_udfs is None or namespace not in _multicolumn_udfs:
         return
 
+    _multicolumn_udfs[namespace] = [
+        udf
+        for udf in _multicolumn_udfs[namespace]
+        if list(udf.udfs.keys())[0] != old_name
+    ]
+
+
+def _register_udfs(config: Optional[LangKitConfig] = None):
+    global _registered
+    if _registered and config is None:
```

Comment:
This could be annoying if they're trying to switch back to the defaults :)

Reply:
they can just pass in the default config right?
`init(config=LangKitConfig())`

Reply:
Yeah, they just need to know they have to pass it in where normally the default `None` gets you the default config...